features/encrypt/distsql/parser/src/main/antlr4/imports/encrypt/RDLStatement.g4

```diff
@@ -44,7 +44,7 @@ resourceName
     ;
 
 encryptColumnDefinition
-    : LP columnDefinition (COMMA plainColumnDefinition)? COMMA cipherColumnDefinition (COMMA assistedQueryColumnDefinition)? (COMMA likeQueryColumnDefinition)? COMMA algorithmDefinition (COMMA algorithmDefinition)? (COMMA algorithmDefinition)? RP
+    : LP columnDefinition (COMMA plainColumnDefinition)? COMMA cipherColumnDefinition (COMMA assistedQueryColumnDefinition)? (COMMA likeQueryColumnDefinition)? COMMA cipherAlgorithm (COMMA assistedAlgorithm)? (COMMA likeAlgorithm)? RP
```

Comment:
In order to be consistent with the definition of [configuration](https://github.com/apache/shardingsphere/blob/master/features/encrypt/api/src/main/java/org/apache/shardingsphere/encrypt/api/config/rule/EncryptColumnRuleConfiguration.java), I think it would be better if they were `encryptAlgorithm`, `assistedQueryAlgorithm` and `likeQueryAlgorithm`.

Reply:
Yes it looks good.